Add Justin Brown to Contributors list

macronym opened the pull request.

Esh07 reviewed (changes requested):
Hey @macronym, Thank you for submitting the pull request! You're almost there. There is a merged conflict that occurred with your PR. 

To ensure your branch is up to date, please follow these simple steps:

1. Go to your Fork https://github.com/macronym/first-contributions/tree/new-branch
2. Click on the 'Sync fork' button to fetch the latest changes from the main branch.
3. Once the sync is complete, click on 'Update branch' to update your branch with the latest changes.

Please resolve the conflict using [this guide](https://github.com/firstcontributions/first-contributions/blob/main/additional-material/git_workflow_scenarios/resolving-merge-conflicts.md) or the [official Github doc](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github).

Once you update, the repository will re-check and merge the PR. Keep up the great work, and feel free to ask any questions. 

Happy coding!
Esh 😊

macronym commented:
I clicked the sync button but there was not update branch button. In bash I could see there was some new contributors and it wouldn't let me retry the push. Not sure what I did wrong.

Esh07 commented:
Hey @macronym, I’m sorry to see that you’re having this issue. Sometimes repositories force changes. I can see that you deleted the branch right after.

Would you mind creating a new PR with the latest changes to `Contributors.md`? Make sure to fetch the latest changes before making any changes to the file.

I hope this will fix the issue and you’ll be able to merge the changes.

Thanks
Esh 😊

macronym commented:
Hey @Esh07 I deleted the folder on my local machine and re-cloned, branched, changed, committed, and pushed and there wasn't any merge conflicts. I wonder what the difference was. In the future, should I do `git fetch` every time immediately after cloning? If not, when should I be fetching?

Esh07 commented:
>  I wonder what the difference was. 

Just to give you a little insight on what actually happened with your last PR: when you made changes, someone in the base repository made changes to the same line where you made your modification. Due to this, Git couldn’t automatically figure out which changes to accept, and it was flagged as a merge conflict. This is described very well in the [GitHub Merge Conflict](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) documentation.

>  In the future, should I do git fetch every time immediately after cloning? If not, when should I be fetching?

When your remote repository (e.g., a GitHub repository) is updated and you want to get those changes, you can perform git fetch or git pull to bring those changes to your local repository. These two commands have slight differences - you can read more about [git fetch vs git pull]((https://www.gitkraken.com/learn/git/problems/git-pull-vs-fetch)) here.

In the context of this repository, your fork was updated with the changes you made and synced with your local changes. 

However, it’s a fork of the base repository (a copy of this repository - first-contribution repository) and it wasn’t updated with the latest changes from this repository while you made your changes. 

This repository is continuously updated as soon as users open PRs and they get merged. 

Therefore, if you make changes to a file and another person makes changes to the same file and at the same line, and their PR gets merged, then your PR will be flagged as a merge conflict because the person who added/deleted/edited the line that you edited is now conflicting.

I hope you get an idea of what actually happened with your PR. 

Thanks
Esh 😊
